src/windowsdesktop/src/sfx/Microsoft.WindowsDesktop.App.Runtime.sfxproj

```diff
@@ -5,7 +5,6 @@
     <PlatformPackageType>RuntimePack</PlatformPackageType>
     <ArchiveName>windowsdesktop-runtime</ArchiveName>
     <InstallerName>windowsdesktop-runtime</InstallerName>
-    <RuntimeIdentifiers>win-x64;win-x86;win-arm64</RuntimeIdentifiers>
```

Comment:
Remove RuntimeIdentifers here which avoids downloads of unnecessary host and targeting packs. This is from an unused multi-rid targeting feature (which doesn't appear to work with R2R anyway). This repo sets an explicit RuntimeIdentifier to avoid multi-targeting.